proc-macros/src/lib.rs

```diff
@@ -99,8 +99,8 @@ pub(crate) mod visitor;
 ///     async fn async_method(&self, param_a: u8, param_b: String) -> u16;
 ///     fn sync_method(&self) -> String;
 ///
-///     // Note that `subscription_sink` and `SubscriptionResult` were added automatically.
-///     fn sub(&self, subscription_sink: SubscriptionResult) -> SubscriptionResult;
+///     // Note that `pending subscription sink` was added automatically.
```

Comment:
are there meant to be underscores instead of spaces in `pending subscription sink`?